### PR TITLE
Basic api calls

### PR DESCRIPTION
**Pull request by kidsbear**
Few basic API calls at the top level, will modify Testrig representation as needed later.

### CODE REVIEW

File: projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgr.java [hunk 1]
```diff
@@ -1024,4 +1024,14 @@ public void uploadTestrig(String containerName, String testrigName, InputStream
     CommonUtil.deleteDirectory(unzipSubdir);
     CommonUtil.delete(zipFile);
   }
+
+  private Path getContainerPath(String containerName) {
+    return Main.getSettings().getContainersLocation().resolve(containerName).toAbsolutePath();
+  }
+
+  public boolean checkContainerExist(String containerName) {
```
Comment (arifogel) [inline projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgr.java]: English grammar: Please rename to checkContainerExists

File: projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgrServiceV2.java [hunk 2]
```diff
@@ -29,40 +32,74 @@
...
+  /** Get information about the Batfish environment and supported methods. */
   @GET
-  @Path("/container/{id}")
-  public Response getContainer(
-      @DefaultValue(DEFAULT_API_KEY) @HeaderParam(SVC_KEY_API_KEY) String apiKey,
-      @PathParam("id") String id) {
-    if (!Main.getAuthorizer().isAccessibleContainer(apiKey, id, false)) {
-      // TODO: put a proper error entity here.
-      return Response.status(Status.NOT_FOUND).build();
+  @Path("/info")
```
Comment (arifogel) [inline projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgrServiceV2.java]: Why are we using string literals here? Should you use an RSC constant?

File: projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgrServiceV2.java [hunk 3]
```diff
@@ -29,40 +32,74 @@
...
+  public Response getInfo() {
```
Comment (arifogel) [inline projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgrServiceV2.java]: I don't like the name of this method. It sounds too generic and not obviously different from e.g. `getStatus`. Please make it more informative. If for instance it is supposed to return information about an `Environment`, then it could be called `getEnvironmentInfo`.

File: projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgrServiceV2.java [hunk 4]
```diff
@@ -29,40 +32,74 @@
...
+    _logger.info("WMS:getInfo\n");
+    try {
+      JSONObject map = new JSONObject();
```
Comment (arifogel) [inline projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgrServiceV2.java]: I thought the whole point of this file was to replace the codehaus JSON provider API. Why is this here again? Was there a complication?

File: projects/coordinator/src/main/java/org/batfish/coordinator/resources/ContainerResource.java [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+    return Response.ok(testrigNames).build();
+  }
+
+  /** Redirect to /testrigs if the user does not supply a testrig ID. */
+  @GET
+  @Path("/testrig")
+  public Response redirectTestrig() {
+    validate();
+    UriBuilder ub = _uriInfo.getBaseUriBuilder();
+    ub.path(CoordConsts.SVC_CFG_WORK_MGR2).path("container").path(_name).path("testrigs");
```
Comment (arifogel) [inline projects/coordinator/src/main/java/org/batfish/coordinator/resources/ContainerResource.java]: `"testrigs"` similarly should be an RSC constant

File: projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgrServiceV2.java (hunk 3 above)
Comment (kidsbear) [inline projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgrServiceV2.java]: I actually copy these two API calls from our previous WorkMgrService. I think I might rename getStatus to something like 'getWorkQueueStatus' and remove getInfo for now. I think the purpose of 'getInfo' is to provide some instructions, but not fully implemented in the previous WorkMgrService.  I think it should be better to add this API call after I got most of other calls set. Sounds ok?

**Comment by kidsbear** [comment]
Trying to start from another direction, see #322 , close for now
